common/workunit/workunit.cpp

```diff
@@ -6308,8 +6308,11 @@ extern WORKUNIT_API IWorkUnitFactory * getWorkUnitFactory()
         {
             const char *forceEnv = getenv("FORCE_DALI_WORKUNITS");
             bool forceDali = forceEnv && !strieq(forceEnv, "off") && !strieq(forceEnv, "0");
-            Owned<IRemoteConnection> env = querySDS().connect("/Environment", myProcessSession(), 0, SDS_LOCK_TIMEOUT);
             IPropertyTree *pluginInfo = NULL;
+#ifdef _CONTAINERIZED
+            pluginInfo = queryComponentConfig().queryPropTree("WorkunitServer");  // MORE - case??
```

Comment:
I think probably would need some additional helm support to add a flag to say "use Cassandra instance X for all workunits" which would then affect all component configurations (probably should be global in fact?)

It's a long time since I looked at this, and it looks like maybe it was not 100% finished...